public/app/features/dashboard/components/PanelEditor/getVisualizationOptions.tsx

```diff
@@ -135,7 +135,15 @@ export function getVisualizationOptions(props: OptionPaneRenderProps): OptionsPa
             );
           };
 
-          return <Editor value={value} onChange={onChange} item={fieldOption} context={context} id={fieldOption.id} />;
+          const dynamicProps: Record<string, any> = {}
+          const { upperBound, lowerBound } = fieldOption?.settings || {}
+          if (upperBound) {
+            createDynamicProps(upperBound, dynamicProps, currentFieldConfig.defaults)
+          }
+          if (lowerBound) {
+            createDynamicProps(lowerBound, dynamicProps, currentFieldConfig.defaults)
```

Comment:
There's no need for this function to mutate `dynamicProps`. I suggest getting rid of the arg and just rely on the returned value.

Reply:
feels like there should be a simpler / less evasive way to complete this, that is also more generic (less specific to min/max bounds) 

Something like
```ts
const dynamicProps = fieldOption.settings.dynamicProps ?  fieldOption.settings.dynamicProps(currentFieldConfig.defaults) : null;
```